tools/projmgr/src/ProjMgr.cpp

```diff
@@ -118,6 +119,7 @@ int ProjMgr::RunProjMgr(int argc, char **argv, char** envp) {
 
   cxxopts::Option solution("s,solution", "Input csolution.yml file", cxxopts::value<string>());
   cxxopts::Option context("c,context", "Input context names [<project-name>][.<build-type>][+<target-type>]", cxxopts::value<std::vector<std::string>>());
+  cxxopts::Option contextReplacement("context-replacement", "Input context replacement name [<project-name>][.<build-type>][+<target-type>]", cxxopts::value<std::string>());
```

Comment:
`contextReplacement` must be also added to `options` ~ line 159 below.
A test case to cover [lines 215-217](https://app.codecov.io/gh/Open-CMSIS-Pack/devtools/pull/1033?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Open-CMSIS-Pack#47b74e14ec088ef960ad1477dd7ab912-R215) would be recommended.